### ISSUE TITLE
extended test flake: expected different manifest version for image pruning

### ISSUE THREAD

**Issue by miminar** [openshift/origin#15452]
```
• Failure [15.436 seconds]
[Feature:ImagePrune] Image prune
/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/images/prune.go:117
  of schema 1
  /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/images/prune.go:75
    should prune old image [It]
    /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/images/prune.go:74

    Expected
        <string>: application/vnd.docker.distribution.manifest.v2+json
    to equal
        <string>: application/vnd.docker.distribution.manifest.v1+json

    /go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/test/extended/images/prune.go:159
```

Due to a recent change in defaulting to accepting manifest schema 2, the test fails because it still expects schema 1 to be the result.

This is already addressed by https://github.com/openshift/origin/pull/14509

Happened in job https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_extended/910/consoleFull#66517459456bf4006e4b05b79524e5923

**Comment by pweil-** [comment]
removing test flake, this is a bug that we didn't update the test

**Comment by miminar** [comment]
Resolved by #14585.
